src/WebJobs.Script/Extensions/ScriptLoggingBuilderExtensions.cs

```diff
@@ -16,16 +15,24 @@ namespace Microsoft.Extensions.Logging
     public static class ScriptLoggingBuilderExtensions
     {
         private static ConcurrentDictionary<string, bool> _filteredCategoryCache = new ConcurrentDictionary<string, bool>();
+        private static ImmutableArray<string> _cachedPrefixes = ScriptConstants.SystemLogCategoryPrefixes;
```

Comment:
Small nit: consider adding a comment or renaming to clarify these are the log categories we want to _allow_ (as opposed to filter out).